service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,129 @@
+            <parameter name="ordersList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name="- placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="Order" displayName="get the person details">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="ordersList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find-one entity-name="mantle.order.OrderHeader" value-field="ordersList">
+                <field-map field-name="orderId"/>
+            </entity-find-one>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderHeaders" displayName="get the grand_total details">
+        <out-parameters>
+            <parameter name="ordersList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <having-econditions>
+                    <econdition field-name="grandTotal" operator="greater" from="50"/>
+                </having-econditions>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderItems" displayName="get the grand_total details">
+        <out-parameters>
+            <parameter name="orderItemList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderItem" list="orderItemList">
+                <econdition field-name="orderId" value="100102"/>
+                <econdition field-name="orderPartSeqId" value="01"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderParts" displayName="get the part_total details">
+        <out-parameters>
+            <parameter name="ordersPartList" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+                <having-econditions>
+                    <econdition field-name="partTotal" operator="less" from="20"/>
+                </having-econditions>
+            </entity-find>
+        </actions>
+    </service>
+
+    <service verb="get" noun="OrderPartData" displayName="get the order part details">
+        <out-parameters>
+            <parameter name="orderPartLists" type="list" required="true"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderPart" list="ordersPartList">
+                <econdition field-name="facilityId" value="ZIRET_WH"/>
+            </entity-find>
+            <set field="orderPartLists" from="[orderId:ordersPartList.orderId
```

Comment:
Please check the output for this service, it should be a list containing individual maps of Order part data.

Reply:
Updated the recommended changes.
Thank you.